graphGen.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/bash
```

Comment:
Use #!/usr/bin/env bash. My distribution doesn't have /usr/bin/bash. See https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang